app/helpers/dashboard_helper.rb

```diff
@@ -14,6 +14,9 @@ def dashboard_actions
   end
 
   def render_widget(widget)
+    if widget.data.include?('trend_id')
+      @trend = Trend.find(widget.data['trend_id'])
```

Comment:
Ideally the dashboard should not know about locals that are used inside the template.
Maybe we should add locals to the description of the widget. Another option is refactoring the template a bit, and using convention variables.
See [http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials](http://guides.rubyonrails.org/layouts_and_rendering.html#using-partials) for more details, specifically 3.4.4